russh/src/server/session.rs

```diff
@@ -919,6 +986,18 @@ impl Session {
         })
     }
 
+    pub fn channel_open_forwarded_streamlocal(
+        &mut self,
+        server_socket_path: &str,
+        client_socket_path: &str,
+    ) -> Result<ChannelId, Error> {
+        self.channel_open_generic(b"forwarded-streamlocal", |write| {
+            // NEED HELP: is this correct?
+            write.extend_ssh_string(server_socket_path.as_bytes());
+            write.extend_ssh_string(client_socket_path.as_bytes());
```

Comment:
Looks like this field is just reserved: https://github.com/openssh/openssh-portable/blob/master/PROTOCOL#L344

Reply:
Pretty sure "socket path" is the client socket path

Reply:
So this should be correct:
```rust
            write.extend_ssh_string(client_socket_path.as_bytes());
            write.extend_ssh_string(b"");
```

Reply:
Probably resolved? I'm kinda confused though since tcp/ip forwarding has both client and server (host/port) but streamlocal forwarding only has client?